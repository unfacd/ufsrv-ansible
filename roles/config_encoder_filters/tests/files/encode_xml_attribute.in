element:
  - ^attr1: val1
  - element1:
    - ^attr3: val3
    - ^attr4: val4
  - ^attr2: val2
