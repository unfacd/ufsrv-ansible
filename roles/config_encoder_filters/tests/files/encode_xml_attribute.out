<element attr1="val1" attr2="val2">
  <element1 attr3="val3" attr4="val4" />
</element>
